doc/format-run.md

```diff
@@ -175,6 +175,22 @@ ordinary index type.
 The type of index in use is metadata that must be known for the whole LSM table
 handle. It does not vary per run.
 
+Independently of the concrete index serialization format, the type and the
+version of the format are stored at the beginning as a 32-bit number called the
+type–version indicator. A type–version indicator has the following shape, where
+bit numbers refer to significance:
+
+* Bits 0–7 contain the version number.
+* Bits 8–15 contain a number denoting the type of the index.
+* Bits 16–31 are zero.
+
+As a result, a type–version indicator can be written in hexadecimal with only
+4-digits of which the first two denote the type and the last two constitute the
+version number. Version numbers are assigned independently for each particular
+type. They must not be zero, which ensures that from the way the type–version
```

Comment:
Does "they" refer to both the version number and type, or one of them?

Reply:
“They” refers to “version numbers”. I think that the natural interpretation is that this occurrence of “they” refers to only one nominal phrase of the previous sentence, and “version numbers” is then the phrase to pick, because it is the only plural phrase in this sentence and is sort of the default because of being the subject phrase. In my opinion, it should be intuitively rather clear that this is meant, but maybe I can clarify this.

Reply:
An improved version is in ee678027a27d290045204b7c75b20cda180865ee.